.circleci/custom-image/install-deps.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+VERSION=3.4.0
```

Comment:
Why are we setting this version var here? It's not used anywhere. Am I missing something?

Reply:
Duh.. I hardcoded it everywhere else.. good catch